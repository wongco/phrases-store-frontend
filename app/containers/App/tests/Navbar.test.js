import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Navbar from '../NavBar';

describe('<Navbar />', () => {
  it('renders without crashing', () => {
    shallow(<Navbar />);
  });

  it('matches snapshot for default criteria', () => {
    const wrapper = shallow(<Navbar />);
    const serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
  });
});
