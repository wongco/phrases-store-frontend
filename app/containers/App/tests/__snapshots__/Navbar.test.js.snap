// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> matches snapshot for default criteria 1`] = `
<div>
  <NavBar__Nav>
    <NavBar__StyledNavLink
      activeStyle={
        Object {
          "color": "rgb(7, 196, 98)",
          "fontWeight": "450",
        }
      }
      exact={true}
      to="/"
    >
      Home
    </NavBar__StyledNavLink>
    <NavBar__StyledNavLink
      activeStyle={
        Object {
          "color": "rgb(7, 196, 98)",
          "fontWeight": "450",
        }
      }
      exact={true}
      to="/add"
    >
      Add
    </NavBar__StyledNavLink>
    <NavBar__StyledNavLink
      activeStyle={
        Object {
          "color": "rgb(7, 196, 98)",
          "fontWeight": "450",
        }
      }
      exact={true}
      to="/phrases"
    >
      Show All
    </NavBar__StyledNavLink>
    <NavBar__Hero>
      The Phrases Store
    </NavBar__Hero>
  </NavBar__Nav>
</div>
`;
